### PR TITLE
[DEV-1805] Add RelatedResources component

### PR DESCRIPTION
**Pull request by MikeAtUqido**
This PR depends on #1020 

#### List of Changes

Add RelatedResources component

#### Motivation and Context

The new webinars sections has a new related resources section

#### How Has This Been Tested?

Manually and Storybbok

#### Screenshots (if appropriate):
Screenshot 2024-07-24 alle 10 26 50

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/components/molecules/RelatedResources/RelatedResources.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import EContainer from '@/editorialComponents/EContainer/EContainer';
+import { Typography } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import { GuideCard, GuideCardProps } from '../GuideCard/GuideCard';
+
+interface RelatedResourcesProps {
+  guide: GuideCardProps;
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/molecules/RelatedResources/RelatedResources.tsx:7]: The component should take an array of GuideCardProps, not a single element and I think you should change the property name to `resources`

File: apps/nextjs-website/src/components/molecules/RelatedResources/RelatedResources.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+}
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/molecules/RelatedResources/RelatedResources.tsx:8]: Without the `downloadableDocuments` property this component is not complete.
You should complete DEV-1806 and possibly add a dependency in this PR

File: apps/nextjs-website/src/components/molecules/RelatedResources/RelatedResources.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const RelatedResources = ({ guide }: RelatedResourcesProps) => {
+  const t = useTranslations('webinar.webinarsSection.relatedResources');
+
+  return (
+    <EContainer>
+      <Typography variant='h4' sx={{ mb: 4, width: '100%' }}>
+        {t('title')}
+      </Typography>
+      <GuideCard {...guide} />
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/molecules/RelatedResources/RelatedResources.tsx:18]: You should map the array here

File: apps/storybook-app/stories/molecules/RelatedResources.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      },
+      imagePath: 'https://via.placeholder.com/2000x1000',
+      link: {
+        href: '#',
+        label: 'Vai alla guida',
+      },
+      mobileImagePath: 'https://via.placeholder.com/150',
+      title: 'Guida pratica',
+    },
+  },
```
Comment (marcobottaro) [inline apps/storybook-app/stories/molecules/RelatedResources.stories.tsx:28]: You should also change this

File: .changeset/few-geese-retire.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": patch
+"storybook-app": patch
```
Comment (marcobottaro) [inline .changeset/few-geese-retire.md:3]: With this PR you are adding a new component, I think it's a new non breaking feature and so a `minor` change, not a `patch`

```suggestion
"nextjs-website": minor
"storybook-app": minor
```

**Review by marcobottaro — CHANGES_REQUESTED** [review]

**Review by MarcoPonchia — APPROVED** [review]

**Comment by MarcoPonchia** [comment]
https://github.com/pagopa/developer-portal/pull/1020

**Pull request closed without merging**